app/assets/stylesheets/colors.scss

```diff
@@ -29,6 +29,8 @@ $sidebars-background: #f0f0f0;
 $sidebars-sub-background: darken($sidebars-background, 12%);
 $left-navbar-drawer-background: darken($sidebars-background, 6%);
 
+$card-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.16), 0 2px 10px 0 rgba(0, 0, 0, 0.12);
```

Comment:
Nope, I will also use it in #6324 [here](https://github.com/diaspora/diaspora/pull/6324/files#diff-b48a3c07783ee2256d72153189c9743bR32). I thought it was more or less a color to me.